pages/common/composer.md

```diff
@@ -0,0 +1,31 @@
+# composer
+
+> Dependency Manager for PHP.
+
+- Interactively create a composer.json file:
+
+`composer init`
+
+- Create new project from an existing package:
+
+`composer create-project {{project_name}} {{target_path}} {{version}}`
+
+- Install packages specified in composer.json file:
```

Comment:
What does "install" mean here? Globally in the local machine? Only within the current project directory? I think it would be nice to make this explicit.